pycalculix/results_file.py

```diff
@@ -1077,6 +1078,16 @@ def _save_node_force(self, line, rfstr, time, mode='force'):
         for (label, val) in zip(labs, vals):
             adict[label] = val
 
+    def _save_node_error(self, line, rfstr, time, mode='error'):
+        """Saves node error"""
+        # [key, node, error]
```

Comment:
Can you please add a comment here describing that the error value
crom the ccx docs? I want the devs to know that the error is a percent difference in strain.